src/test/java/com/Query/query/Controller/AlunoControllerTest.java

```diff
@@ -19,7 +19,7 @@
 	private MockMvc mockMvc;
 
 	@Test
-	private void retornaAlunoStatus200paraUmParametro() throws Exception {
+	public void retornaAlunoStatus200paraUmParametro() throws Exception {
 
 		URI uri = new URI("/aluno/1");
 

```

Comment:
Persistir um aluno com id 1 direto no código para realizar o teste, pois falhou aqui na minha máquina. Realizar essa alteração em todas as classes de teste (professor, disciplina e turma)

```suggestion
Aluno aluno = montaObjetoAluno(dadosAluno);
repository.save(aluno);
```

Reply:
feito